### DISCUSSION TITLE
Model Logistic Regression menghasilkan informasi NA

### DISCUSSION THREAD

**Discussion by Saupia** [teamalgoritma/community#458] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/114889462/202788458-aadda4f3-b266-4bec-9ce9-746abecfa7e0.png)

Mohon penjelasannya kenapa modelnya menunjukkan angka NA seperti ini ya? ketika aku buat semua variable. 
Sedangkan kalau hanya 1 variabel prediktor nilainya keluar. 
![image](https://user-images.githubusercontent.com/114889462/202788774-2a327c56-1ea1-4ce8-829a-2c439144d6e3.png)

**Comment by divakartika** [comment, marked as answer]
Dear @Saupia 

Pada hasil summary tersebut dapat dilihat bahwa NA terjadi akibat singularity
![image](https://user-images.githubusercontent.com/79572421/202957421-c4af2744-5126-4193-8278-d77944c12f40.png)

Singularity merupakan kasus ekstrim dari multicollinearity, dengan kata lain hal ini terjadi akibat korelasi antar prediktor yang terlalu kuat. Dapat dilakukan cek korelasi dengan ggcorr dan uji vif terlebih dahulu untuk mengetahui prediktor apa saja yang berkorelasi kuat dan hapus prediktor-prediktor yang tidak diperlukan. Umumnya prediktor yang multicolinear bersifat berpasangan, sehingga kita hanya perlu menghapus salah satunya.

Referensi:
- https://www.statistics.com/glossary/singularity/
- https://discuss.analyticsvidhya.com/t/what-is-the-singularity-error-in-linear-regression/3924

**Reply by Saupia** [reply]
Terima kasih.
